packages/react/__tests__/__snapshots__/exports.ts.snap

```diff
@@ -111,6 +111,7 @@ exports[`@aws-amplify/ui-react/internal exports should match snapshot 1`] = `
   "IconVisibilityOff",
   "IconWarning",
   "PrimitiveCatalog",
+  "STORAGE_BROWSER_ICON_PATHS",
```

Comment:
Should not be a public export

Reply:
Removed from snapshot after changing export location

Reply:
I had thought that I saw this test pass, but it is failing again. It seems like this test is confirming internal exports though, so I think it being in this snapshot should be correct? Or am I missing something?